src/components/BookList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BookList = props => {
+  const { books } = props;
+
+  return (
+
+    <div className="booklist">
+
+      <table>
+        <tr>
+          <th>Book Id</th>
+          <th>Category</th>
+          <th>Title</th>
+        </tr>
+        { books.map(elem => (
+          <Book
+            key={elem.id}
+            title={elem.title}
+            category={elem.category}
+            id={elem.id}
+          />
+        ))}
+      </table>
+    </div>
+
+  );
+};
+
+BookList.propTypes = {
+
+  books: PropTypes.arrayOf(PropTypes.object).isRequired
```

Comment:
- it should connects to Redux store and fetches books from the state [[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png) ]
- it should use `mapStateToProps` to fetch books from the `store`